### ISSUE TITLE
Is the plugin thread safe?

### ISSUE THREAD

**Issue by garagatyi** [coveo/fmt-maven-plugin#16]
We are using the plugin in Eclipse Che project. When I'm building it in parallel mode maven shows next warning:
<details>
<summary>Unfold</summary>

```
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but project      *
[WARNING] * contains the following plugin(s) that have goals not marked   *
[WARNING] * as @threadSafe to support parallel building.                  *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against maven-core                              *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in Che Selenium :: Core:
[WARNING] com.coveo:fmt-maven-plugin:1.8.0
[WARNING] Enable debug to see more precisely which goals are not marked @threadSafe.
[WARNING] *****************************************************************
```
</details>
Is the plugin thread safe? Can the warning be fixed?

**Comment by GuiSim** [comment]
I'm not super familiar with parallel builds in Maven.
Would the plugin be invoked on the same files in multiple concurrent threads/processes ?

**Comment by garagatyi** [comment]
Parallel threads are invoked on different maven modules of a multi-module project. I don't see any valid case when these threads may interfere on sources formatting/checking.

**Comment by GuiSim** [comment]
Thanks for the answer! 
It sounds to me that it is safe to add this annotation.

Would you be willing to do a PR with this change?

**Comment by garagatyi** [comment]
I would like but don't have spare time to do this in this or the following months

**Comment by malaporte** [comment]
Fixed by #17

**Comment by garagatyi** [comment]
Yeah, it was really small fix. Tnx
